prometheus.yaml

```diff
@@ -0,0 +1,10 @@
+global:
+  scrape_interval:     5s # Default is every 1 minute.
+  evaluation_interval: 5s # Default is every 1 minute.
+  
+scrape_configs:
+  - job_name: prometheus
+    honor_timestamps: true
+    scheme: http
+    static_configs:
+    - targets: ['otel-collector:8889'] # using the name of the OpenTelemetryCollector container defined in the docker compose file
```

Comment:
@0xkr8os Isn't the otel collector on port 4317

Reply:
for receiving data yes, but the endpoint which exposes prometheus metrics is defined at 0.0.0.0:8889 in the otel-collector